app/views/users/show.haml

```diff
@@ -46,6 +46,11 @@
         = link_to search_posts_path(author_id: [@user.id], commit: 'Search', completed: true), rel: 'nofollow' do
           = image_tag "icons/book.png", class: 'vmid', alt: ''
           Completed Posts
+    %tr
+      %td.centered{class: cycle('even', 'odd')}
+        = link_to search_bookmarks_path(user_id: @user.id, commit: 'Search'), rel: 'nofollow' do
```

Comment:
Should we only show this if the user's bookmarks are public?

Reply:
Maybe? I thought about that but decided against it because it felt unaesthetic to me for the user infoboxes themselves to change that way when nothing else on them does.

Reply:
Ppl seem to prefer that private bookmarks be indistinguishable from public-but-zero bookmarks